cmd/cronjob-checker/README.md

```diff
@@ -0,0 +1,88 @@
+## CronJob Event Checker
+
+The cronjob-checker reaches out to a specified namespace and retrieves events from cronjobs. It then ranges over the events for a specified reason. If there is an event with the specified reason it will alert kuberhealthy.
+
+You can specify the namespace to check with the `NAMESPACE` environment variable in the `.yaml` file.
+
+You can specify the event reason to check with the `REASON` environment variable in the `.yaml` file.
+
+The check will exit if no REASON is given in the yaml file.
+
+The check will exit if no NAMESPACE is given in the yaml file.
+
+The check will exit if it is unable to retrieve events from the cronjob in a given namespace.
+
+All namespace fields will need to be updated to applied namespace to give permissions appropriately to the check.
+
+#### Example CronJob Event Checker
+
+```yaml
+---
+apiVersion: comcast.github.io/v1
+kind: KuberhealthyCheck
+metadata:
+  name: cronjob-checker
+  namespace: kuberhealthy
+spec:
+  runInterval: 1h
+  timeout: 10m
+  podSpec:
+    serviceAccountName: cronjob-checker
+    containers:
+      - name: cronjob-checker
+        image: kuberhealthy/cronjob-checker:v1.1.0
+        imagePullPolicy: IfNotPresent
+        env:
+          - name: NAMESPACE
```

Comment:
True this up with the `fieldRef` in your example yaml.

Reply:
done